Tests/iaas/scs-0104-v1.1-images.yaml

```diff
@@ -0,0 +1,30 @@
+images:
+- name: "Ubuntu 24.04"
+  source:
+  - https://cloud-images.ubuntu.com/releases/noble/
+  - https://cloud-images.ubuntu.com/noble/
+  status: mandatory
+- name: "Ubuntu 22.04"
+  source:
+  - https://cloud-images.ubuntu.com/releases/jammy/
+  - https://cloud-images.ubuntu.com/jammy/
+- name: "ubuntu-capi-image"
+  name_scheme: "ubuntu-capi-image v[0-9]\\.[0-9]+(\\.[0-9]+)?"
+  source: https://swift.services.a.regiocloud.tech/swift/v1/AUTH_b182637428444b9aa302bb8d5a5a418c/openstack-k8s-capi-images/ubuntu-2204-kube
+  status: recommended
+- name: "Ubuntu 20.04"
+  source:
+  - https://cloud-images.ubuntu.com/releases/focal/
+  - https://cloud-images.ubuntu.com/focal/
+- name: "Debian 12"
+  source:
+  - https://cloud.debian.org/images/cloud/bookworm/
+  - https://cdimage.debian.org/cdimage/cloud/bookworm/
+- name: "Debian 11"
+  source:
+  - https://cloud.debian.org/images/cloud/bullseye/
+  - https://cdimage.debian.org/cdimage/cloud/bullseye/
+- name: "Debian 10"
+  source:
+  - https://cloud.debian.org/images/cloud/buster/
+  - https://cdimage.debian.org/cdimage/cloud/buster/
```

Comment:
I think it can be problematic if an environment has Debian 10, but NOT 11 or 12, as seems to be the case with Syseleven, see #696. Then users have a hard time choosing a current version. They can always upload one, I guess, but that is an additional barrier.

Reply:
I would expect Debian 12 to be mandatory at this point, same for 11 and 10 recommended.

The whole point of requiring a minimal set of public images is to ensure that customers do not have to upload their own stock images and remember to update them once per month ...
This is certainly a good answer to folks that want specific images. But mainstream images are better provided by the cloud operator, as this is more efficient for everyone.

Reply:
Then we shall do that for v5 of SCS-compatible IaaS